cron/closeIndices.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+# set this to run as a cronjob (after backup has completed)
+# to regularly remove indexes
+
+# .conf file will determine what indexes are operated on
+# Create a starter .conf file with backupDiscover.py
+
+import sys
+import logging
+from datetime import datetime, date, timedelta
+from configlib import getConfig, OptionParser
+from logging.handlers import SysLogHandler
+
+from mozdef_util.utilities.toUTC import toUTC
+from mozdef_util.elasticsearch_client import ElasticsearchClient
+
+
+logger = logging.getLogger(sys.argv[0])
```

Comment:
We can use the mozdef_util version of logger (https://github.com/mozilla/MozDef/blob/master/mozdef_util/mozdef_util/utilities/logger.py)

like `from mozdef_util.utilities.logger import logger`

Reply:
ack will revise.